server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -154,6 +154,9 @@ export function verifyStorageToken(
         try {
             await tenantManager.verifyToken(claims.tenantId, token);
         } catch (error) {
+            if (error instanceof NetworkError) {
```

Comment:
On the client, I've been trained not to trust `instanceof`, it may be due to fragility around multiple bundles or something, unsure.  If you write a type guard for `NetworkError` like I mentioned above, you could use that -- or maybe `instanceof` is safe on the server in which case party on!

Reply:
Also - why different approaches here v. in alfred/index.ts for inspecting the error thrown by `tenantManager.verifyToken`?

Reply:
Good point, I copied this approach from a couple lines above. Will update all to use a type guard